### PR TITLE
Makefile.am: Use @LIBS@ in LDADD to get -lflint-arb on Debian

### PR DESCRIPTION
**Pull request by mkoeppe**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -20,7 +20,7 @@ check_PROGRAMS =
 dist_doc_DATA =
 
 # TODO: we currently build all tests with linked c++ libraries
-LDADD = libeanticxx.la libeantic.la -larb -lflint -lgmpxx -lgmp
+LDADD = libeanticxx.la libeantic.la @LIBS@ -lgmpxx -lgmp
```
Comment (saraedum) [inline Makefile.am:23]:
```suggestion
LDADD = libeanticxx.la libeantic.la @LIBS@ -lgmpxx
```

**Comment by videlec** [comment]
thx

File: Makefile.am (hunk 1 above)
Comment (mkoeppe) [reply Makefile.am:23]: For static builds, must have -lgmp after -lgmpxx

**Comment by mkoeppe** [comment]
Can we get a new release please?

**Comment by mkoeppe** [comment]
It could be called 0.1.8

**Pull request merged**